tests/async.rs

```diff
@@ -93,24 +93,21 @@ where
 
     fn test<C, F>(self, c: C)
     where
-        F: IntoFuture<Item = (), Error = mysql_async::errors::Error>,
+        F: IntoFuture<Item = (), Error = mysql_async::error::Error>,
         C: FnOnce(mysql_async::Conn) -> F,
     {
         let listener = net::TcpListener::bind("127.0.0.1:0").unwrap();
         let port = listener.local_addr().unwrap().port();
+
```

Comment:
Can we cut this just to minimize the diff? Sorry for the nitpick :)

Reply:
No problem. Is there somewhere I can find the rustfmt config that you are using? I tried a rustfmt and got a lot of changes, so I think I must have a different setup. If you have one I can use that would help keep this and future pull requests nice and tidy.

Reply:
Ah, I probably last formatted it with a much older version. I just pushed to `master` with the latest rustfmt applied, so maybe try merging and then run rustfmt again?